src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -0,0 +1,339 @@
+                />
+              ),
+              fixed: 'events chart',
+            })}
+        </HeaderBox>
+        <ContentColumn>
+          <EventInterfaces event={event} projectId={this.projectId} />
+        </ContentColumn>
+        <SidebarColumn>
+          {event.groupID && (
+            <LinkedIssuePreview groupId={event.groupID} eventId={event.eventID} />
+          )}
+          {event.type === 'transaction' && (
+            <RelatedEvents
+              organization={organization}
+              event={event}
+              location={location}
+              eventView={eventView}
+            />
+          )}
+          <EventMetadata event={event} eventJsonUrl={eventJsonUrl} />
+          <SidebarBlock>
+            <TagsTable tags={event.tags} />
+          </SidebarBlock>
+        </SidebarColumn>
+      </ColumnGrid>
+    );
+  }
+
+  renderError(error) {
+    const notFound = Object.values(this.state.errors).find(
+      resp => resp && resp.status === 404
+    );
+
+    if (notFound) {
+      return this.renderWrapper(<NotFound />);
+    }
+
+    return this.renderWrapper(super.renderError(error, true, true));
+  }
+
+  renderLoading() {
+    return this.renderWrapper(super.renderLoading());
+  }
+
+  renderWrapper(children: React.ReactNode) {
+    const {organization, location} = this.props;
+    const eventView = this.getEventView();
+    const {event} = this.state;
+
+    return (
+      <EventDetailsWrapper
+        organization={organization}
+        location={location}
+        eventView={eventView}
+        event={event}
+      >
+        {children}
+      </EventDetailsWrapper>
+    );
+  }
+}
+
+type EventDetailsWrapperProps = {
+  organization: Organization;
+  location: Location;
+  eventView: EventView;
+  event: Event | undefined;
+  children: React.ReactNode;
+};
+
+class EventDetailsWrapper extends React.Component<EventDetailsWrapperProps> {
+  getDocumentTitle = (): string => {
+    const {event, eventView} = this.props;
+
+    const titles = [t('Discover')];
+
+    const eventViewName = eventView.name;
+    if (typeof eventViewName === 'string' && String(eventViewName).trim().length > 0) {
+      titles.push(String(eventViewName).trim());
+    }
+
+    const eventTitle = event ? getTitle(event).title : undefined;
+
+    if (eventTitle) {
+      titles.push(eventTitle);
+    }
+
+    return titles.join(' - ');
+  };
+
+  render() {
+    const {organization, location, eventView, event, children} = this.props;
+
+    return (
+      <DocumentTitle title={`${this.getDocumentTitle()} - ${organization.slug} - Sentry`}>
+        <React.Fragment>
+          <PageHeader>
+            <DiscoverBreadcrumb
+              eventView={eventView}
+              event={event}
+              organization={organization}
+              location={location}
+            />
+          </PageHeader>
+          {children}
+        </React.Fragment>
+      </DocumentTitle>
+    );
+  }
+}
+
+const EventHeader = props => {
```

Comment:
Does `props` need a type here?

Reply:
fixed in https://github.com/getsentry/sentry/pull/15562/commits/4090f54ce30e6b01d094db1840edf759b3550550